data/5e/wizards_of_the_coast/core/players_handbook/ph_races.lst

```diff
@@ -1,49 +1,49 @@
-Sprite		SIZE:T	MOVE:Walk,0	BONUS:VAR|RaceBase_Walk|10	BONUS:VAR|RaceBase_Fly|40						BONUS:STAT|STR|-8	BONUS:STAT|DEX|8	BONUS:STAT|CON|0	BONUS:STAT|INT|4	BONUS:STAT|WIS|2	BONUS:STAT|CHA|0																				ABILITY:Internal|AUTOMATIC|Racial Traits ~ Sprite																																																																					MONSTERCLASS:Hit Die:1				RACETYPE:Fey							CR:1/4		SOURCEPAGE:p.310	KIT:2|Default Creature ~ Sprite|NG
-Tiger			SIZE:L	MOVE:Walk,0	BONUS:VAR|RaceBase_Walk|40							BONUS:STAT|STR|6	BONUS:STAT|DEX|4	BONUS:STAT|CON|4	BONUS:STAT|INT|-8	BONUS:STAT|WIS|2	BONUS:STAT|CHA|-2																				ABILITY:Internal|AUTOMATIC|Racial Traits ~ Tiger																																																																					MONSTERCLASS:Hit Die:5				RACETYPE:Beast							CR:1			SOURCEPAGE:p.311	KIT:2|Default Creature ~ Tiger|UA
-Warhorse		SIZE:L	MOVE:Walk,0	BONUS:VAR|RaceBase_Walk|60							BONUS:STAT|STR|8	BONUS:STAT|DEX|2	BONUS:STAT|CON|2	BONUS:STAT|INT|-8	BONUS:STAT|WIS|2	BONUS:STAT|CHA|-4																				ABILITY:Internal|AUTOMATIC|Racial Traits ~ Warhorse																																																																					MONSTERCLASS:Hit Die:3				RACETYPE:Beast							CR:1/2		SOURCEPAGE:p.311	KIT:2|Default Creature ~ Warhorse|UA
-Wolf			SIZE:M	MOVE:Walk,0	BONUS:VAR|RaceBase_Walk|40							BONUS:STAT|STR|2	BONUS:STAT|DEX|4	BONUS:STAT|CON|2	BONUS:STAT|INT|-8	BONUS:STAT|WIS|2	BONUS:STAT|CHA|-4			BONUS:VAR|AC_Natural_Armor|1|TYPE=Base											ABILITY:Internal|AUTOMATIC|Racial Traits ~ Wolf																																																																						MONSTERCLASS:Hit Die:2				RACETYPE:Beast	RACESUBTYPE:RangerCompanion		CR:1/4		SOURCEPAGE:p.311	KIT:2|Default Creature ~ Wolf|UA
-Zombie		SIZE:M	MOVE:Walk,0	BONUS:VAR|RaceBase_Walk|20							BONUS:STAT|STR|2	BONUS:STAT|DEX|-4	BONUS:STAT|CON|6	BONUS:STAT|INT|-8	BONUS:STAT|WIS|-4	BONUS:STAT|CHA|-6			BONUS:VAR|DarkvisionRange|60													ABILITY:Internal|AUTOMATIC|Racial Traits ~ Zombie																																																																					MONSTERCLASS:Hit Die:3				RACETYPE:Undead							CR:1/4		SOURCEPAGE:p.311	KIT:2|Default Creature ~ Zombie|NE
+# CVS $Revision: $ $Author: $ -- Mon Feb 22 22:56:28 2016 -- reformated by PCGen PrettyLST v6.06.00																					
+																					
+# Basic Rules Races																					
+# Race Name	SORTKEY	Size	Move	Stat bonus	Ability	Nb Legs	Nb Hands	Main Race Type	Race Subtype	Type	Source Page										
+Dwarf	SORTKEY:Base_PC	SIZE:M	MOVE:Walk,25	BONUS:STAT|CON|2	ABILITY:Internal|AUTOMATIC|Racial Traits ~ Dwarf	LEGS:2	HANDS:2	RACETYPE:Humanoid	RACESUBTYPE:Dwarf	TYPE:Humanoid.PC	SOURCEPAGE:p.18										
```

Comment:
Why are you making the upcoming conversion process harder? If you are volunteering to handle the entire 5e conversion to the new pcgen formula system, then feel free to remove these intermediate steps to making that easier for me. Arbitrarily removing code without understanding the reasoning is not a good practice though.